src/AlternativesPerformance.h

```diff
@@ -1,28 +1,34 @@
 #ifndef AlternativesPerformance_H
 #define AlternativesPerformance_H
 
+#include "Category.h"
 #include "Criteria.h"
 #include "Performance.h"
 #include "PerformanceTable.h"
 #include <iostream>
 #include <iterator>
-#include <map>
 #include <ostream>
 #include <string.h>
+#include <unordered_map>
 #include <vector>
 
+extern Category default_cat;
```

Comment:
nit: I think it's a good practice to put default value as const and use a naming all in caps letters (DEFAULT_MAP)